src/common/extension-telemetry-events.ts

```diff
@@ -35,28 +35,29 @@ export const START_OVER_TEST: string = 'startOverTest';
 export const CANCEL_START_OVER_TEST: string = 'cancelStartOverTest';
 export const START_OVER_ASSESSMENT: string = 'startOverAssessment';
 export const CANCEL_START_OVER_ASSESSMENT: string = 'cancelStartOverAssessment';
-export const START_OVER_MEDIUM_PASS: string = 'startOverMediumPass';
-export const CANCEL_START_OVER_MEDIUM_PASS: string = 'cancelStartOverMediumPass';
+export const START_OVER_MEDIUM_PASS: string = 'startOverQuickAssess';
```

Comment:
The variables still refer to medium pass here.

Reply:
another good catch! thank you!